lib/matplotlib/backends/gtk3_compat.py

```diff
@@ -0,0 +1,34 @@
+import matplotlib
+
+error_msg_gtk3 = "Gtk3 backend requires the installation of pygobject or pgi."
+
+# Import the first library that works from the rcParam list
+# throw ImportError if none works
+for lib in matplotlib.rcParams['backend.gi_preference']:
+    try:
+        gi = __import__(lib, globals(), locals(), [], 0)
+        break
+    except ImportError:
+        pass
+else:
+    raise ImportError(error_msg_gtk3)
+
+# Check version
+try:
+    gi.require_version("Gtk", "3.0")
+except AttributeError:
+    raise ImportError(
+        "pygobject version too old -- it must have require_version")
```

Comment:
Can you add the minimum required version here? It will be easier for a user if the errors specify which min version is supported.